src/AWSCore.jl

```diff
@@ -203,7 +203,10 @@ Service endpoint URL for `request`.
 function service_url(aws, request)
     endpoint = get(request, :endpoint, request[:service])
     region = "." * aws[:region]
-    if endpoint == "iam" || (endpoint == "sdb" && region == ".us-east-1") || region=="."
+
+    regionless_service_endpoints = ["iam", "route53"]
+
+    if endpoint in regionless_service_endpoints || (endpoint == "sdb" && region == ".us-east-1") || region=="."
```

Comment:
Whitespace and name change:
```suggestion
    if endpoint in regionless_endpoints || (endpoint == "sdb" && region == ".us-east-1") || region == "."
```